asset/quickstart/get-saved-query/getSavedQuery.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+// [START asset_quickstart_get_saved_query]
+ 
+package GetSavedQueryFunction
+ 
+import (
+	"context"
+	"fmt"
+	"strconv"
+ 
+	asset "cloud.google.com/go/asset/apiv1"
+	"cloud.google.com/go/asset/apiv1/assetpb"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+)
+ 
+func getSavedQuery(projectId, savedQueryID string) error {
```

Comment:
We can make this easier for testing: pass in a `io.Writer` object into the function, like [here](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/dlp/snippets/inspect/inspect_string.go#L28). Then, when you print, call `fmt.Fprintf()` instead of `Println()`.

Reply:
according to the style guide, we expect clients to copy and past these code: https://googlecloudplatform.github.io/samples-style-guide/#no-cli

if thats the expectation, should we try to avoid using io.Writer so that clients dont have to modify anything before using the code sample?

Reply:
You are 100% correct. That said, the Golang sample style (currently) is to use `io.Writer()` for the purposes of testing. 

(FWIW we are between consolidating global Google Cloud samples style and individual language styles. This is one of the places where the two style guides differ. I apologize for the confusion.)